### PR TITLE
Update for India mobile patterns

### PR DESCRIPTION
**Pull request by alecx**
Updated India mobile patterns based on
http://www.dot.gov.in/sites/default/files/u10/MSC%20Code%20as%20on%2011052015%20%284%29_0.xls

Note: I have only added missing prefixes and did not delete any of the
few that are not listed on the website.

### CODE REVIEW

**Comment by andyst** [comment]
Thanks for the report! We're going to review the changes, also consider the updates reported at http://dot.gov.in/sites/default/files/u10/Instruction%20dated%2013.04.2015.pdf and update the metadata accordingly.

**Comment by raincoatforever** [comment]
Hey guys , 
Is there an approx timeline as to when this PR might go in ? 
I'd like to use this asap :)

**Comment by andyst** [comment]
It's on our radar and my guess is that it's going to be in a release in the next weeks. It's not in the upcoming release (this or early next week). There are many metadata changes right now that have to be processed.

**Comment by raincoatforever** [comment]
@andyst : Thanks for the quick response . Just wanted to check if there has been any update. Is this any closer to the release ?  Any ETA ? 

Thanks !

**Comment by raincoatforever** [comment]
hey @andyst , just a gentle reminder . Any updates regarding this ?

**Comment by raincoatforever** [comment]
Hey guys , any updates here ?

**Comment by keghani** [comment]
Hi!

We just released 7.0.9 which covers some of your changes. Can you please rebase? Then the outstanding changes changes will be clearer to understand and easier to verify.

File: resources/PhoneNumberMetadata.xml [hunk 1]
```diff
@@ -10731,7 +10731,7 @@
                 9[7-9]
               )|
               6(?:
-                0[0127]|
```
Comment (onecityuni) [inline resources/PhoneNumberMetadata.xml]: 0161

**Comment by keghani** [comment]
India metadata is up-to-date in release 7.0.11.

**Pull request closed without merging**